src/components/Alerting.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { FC, useState, useContext } from 'react';
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { Input, Label, Select, useStyles } from '@grafana/ui';
+import { Collapse } from './Collapse';
+import { Controller, useFormContext } from 'react-hook-form';
+import { ALERTING_SEVERITY_OPTIONS, TIME_UNIT_OPTIONS } from './constants';
+import { AlertRule } from 'types';
+import { InstanceContext } from './InstanceContext';
+import { AlertAnnotations } from './AlertAnnotations';
+import { AlertLabels } from './AlertLabels';
+
+interface Props {
+  alertRules: AlertRule[];
+  editing: boolean;
+  checkId?: number;
+}
+
+const getStyles = (theme: GrafanaTheme) => ({
+  subheader: css`
+    margin-top: ${theme.spacing.md};
+  `,
+  link: css`
+    text-decoration: underline;
+  `,
+  container: css`
+    background: #202226;
+    padding: ${theme.spacing.md};
+    display: flex;
+    flex-direction: column;
+  `,
+  inputWrapper: css`
+    margin-bottom: ${theme.spacing.md};
+  `,
+  numberInput: css`
+    max-width: 72px;
+    margin-right: ${theme.spacing.sm};
+  `,
+  horizontallyAligned: css`
+    display: flex;
+    align-items: center;
+  `,
+  text: css`
+    font-size: ${theme.typography.size.sm};
+    color: ${theme.colors.formLabel};
+  `,
+  select: css`
+    max-width: 200px;
+  `,
+  severityContainer: css`
+    margin-bottom: ${theme.spacing.md};
+  `,
+});
+
+export const Alerting: FC<Props> = ({ alertRules, editing, checkId }) => {
+  const [showAlerting, setShowAlerting] = useState(false);
+  const { instance } = useContext(InstanceContext);
+  const { register } = useFormContext();
+  const styles = useStyles(getStyles);
+
+  if (!instance.alertRuler) {
+    return (
+      <Collapse label="Alerting" onToggle={() => setShowAlerting(!showAlerting)} isOpen={showAlerting} collapsible>
+        <div className={styles.container}>
+          <p>
+            Synthetic Monitoring uses &nbsp;
+            <a href="https://grafana.com/docs/grafana-cloud/alerts/grafana-cloud-alerting/" className={styles.link}>
+              Grafana Cloud Alerting
+            </a>
+            &nbsp; for creating alert rules. You can edit this check in &nbsp;
+            <a href="https://grafana.com" className={styles.link}>
+              Grafana Cloud
+            </a>
+            &nbsp; to add alerts.
+          </p>
+        </div>
+      </Collapse>
+    );
+  }
+
+  if (alertRules.length && editing) {
```

Comment:
The reasoning there is that it will be hard to translate a promQL expression back into the UI flow we have right now. It's also possible the rule will get edited in alerting ui into something that _can't_ be translated back into our UI controls. So for now we're just going to allow creation and deletion, and push folks to use alerting-ui for all the management